### PR TITLE
Swap width/height in GetWinsize and monitorTtySize

### PR DESCRIPTION
**Pull request by ahmetb**
Fixing code lost in translation

Signed-off-by: Ahmet Alp Balkan ahmetalpbalkan@gmail.com
Emergency cc: @icecrime @jfrazelle @calavera @brendandixon

### CODE REVIEW

**Comment by jfrazelle** [comment]
you rule

**Comment by jfrazelle** [comment]
omg its beautiful LGTM

**Comment by jfrazelle** [comment]
ping @icecrime @tiborvass

**Comment by icecrime** [comment]
Testing!

**Comment by jfrazelle** [comment]
I tried ls, vim , top and now im trying nyancat in conemu

**Comment by jfrazelle** [comment]
btw i put the binary here if you wanna try  http://jesss.s3.amazonaws.com/tmp/docker-1.5.0-dev.exe

**Comment by jfrazelle** [comment]
nyan

docker run --rm -it supertest2014/nyan

File: pkg/term/term_windows.go [hunk 1]
```diff
@@ -48,8 +48,8 @@ func GetWinsize(fd uintptr) (*Winsize, error) {
 
 	// TODO(azlinux): Set the pixel width / height of the console (currently unused by any caller)
 	return &Winsize{
-		Width:  uint16(info.Window.Bottom - info.Window.Top + 1),
-		Height: uint16(info.Window.Right - info.Window.Left + 1),
+		Width:  uint16(info.Window.Right - info.Window.Left + 1),
+		Height: uint16(info.Window.Bottom - info.Window.Top + 1),
```
Comment (icecrime) [inline pkg/term/term_windows.go]: I think those `+ 1` are wrong: if I run an interactive session and sleep on the `a` key, it wraps one character "too late", and gives me empty lines. Behavior is correct with the `+1`.

Comment (ahmetalpbalkan) [inline pkg/term/term_windows.go]: cc: @brendandixon @jhowardmsft about this off by one

**Comment by icecrime** [comment]
Question regarding the `+ 1` offset, otherwise LGTM :+1:

File: pkg/term/term_windows.go (hunk 1 above)
Comment (jhowardmsft) [inline pkg/term/term_windows.go]: That wrapping should be solved by https://github.com/Microsoft/docker/commit/7c83ec1ec94814f771b2d3c44519e93d5de51f29 which @brendandixon or @ahmetalpbalkan where going to open a PR for.

**Pull request merged**